crates/primitives/src/utils.rs

```diff
@@ -1,17 +1,46 @@
 use crate::bits::FixedBytes;
-use tiny_keccak::{Hasher, Keccak};
+
+#[cfg(all(feature = "native-keccak", not(feature = "tiny-keccak")))]
+#[link(wasm_import_module = "vm_hooks")]
+extern "C" {
+    /// When targeting VMs with native keccak hooks, the `native-keccak` feature
```

Comment:
nit: docstring should specify that `tiny_keccak` must be disabled in order for this to work

Reply:
Added documentation and addressed @DaniPopes's comments :)

I had to add `cfg_if` as a new dependency btw. I went ahead and made it part of the workspace